notifications/README.md

```diff
@@ -0,0 +1,49 @@
+## Notifications
+
+The `notifications` package is a client library for the [weaveworks/notifications](https://github.com/weaveworks/notification) service.
+
+
+```golang
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/weaveworks/common/notifications"
+)
+
+func main() {
+	url := "http://eventmanager.notification.svc.cluster.local.:80"
+	sender := notifications.CreateSender(url)
+
+	instanceID := "1"
+	instanceName := "super-cool-instance"
+
+	// Format your message according to the different channels to which it will be sent
+	text := fmt.Sprintf("Instance %s has exploded", instanceName)
+	html := fmt.Sprintf("Instance <b>%s</b> has exploded", instanceName)
+	markdown := fmt.Sprintf("Instance _%s_ has exploded", instanceName)
+
+	timestamp := time.Now()
+
+	message := notifications.Message{
+		Browser: notifications.BrowserMessage{
+			Type:      "critical",
+			Text:      text,
+			Timestamp: timestamp,
+		},
+		Email: notifications.EmailMessage{
+			Subject: "Explosion!",
+			Body:    html,
+		},
+		Slack: notifications.SlackMessage{
+			Text: markdown,
+		},
```

Comment:
What if a client wants to see `timestamp` and `type` in `slack` and `email` as well?

Reply:
They have to embed that in the message body. The browser has special logic for handling different event types and timestamps. 

The equivalent would be to specify HTML with the correct inline styling or class names in the message in order for the notification to be styled correctly. That puts too much strain/work on the user of this API.